cmake/pcre2-config.cmake.in

```diff
@@ -5,11 +5,16 @@
 #
 # Static vs. shared
 # -----------------
-# To make use of the static library instead of the shared one, one needs
-# to set the variable PCRE2_USE_STATIC_LIBS to ON before calling find_package.
+# To make use of the static library instead of the shared one if both have been
+# built, one needs to set the variable PCRE2_USE_STATIC_LIBS to ON before calling find_package.
+# PCRE2_USE_STATIC_LIBS will be ignored if only the static or the shared library has been built.
```

Comment:
I'm unsure if `ignore` is the desired behavior if `PCRE2_USE_STATIC_LIBS` is set. (Vcpkg wants it because the triplet determines the linkage.) In general, CMake config mode would be able to continue search for config in other locations - and it might find another one which provides static libs.

Reply:
Fixed. If `PCRE2_USE_STATIC_LIBS` is set, the shared libraries will not be imported.